core/src/main/java/hivemall/LearnerBaseUDTF.java

```diff
@@ -85,6 +87,8 @@ protected Options getOptions() {
         opts.addOption("loadmodel", true, "Model file name in the distributed cache");
         opts.addOption("dense", "densemodel", false, "Use dense model or not");
         opts.addOption("dims", "feature_dimensions", true, "The dimension of model [default: 16777216 (2^24)]");
+        opts.addOption("mini_batch", false, "Use mini batch algorithm or not");
+        opts.addOption("mini_batch_ratio", true, "The mini batch sampling ratio against all dataset");
```

Comment:
It might be better to consider also providing mini-batch size explicitly because expected/recommended mini-batch size is 100 or so.

A design issue is that providing both of them or not.